test/wanda_web/views/execution_view_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule WandaWeb.ExecutionViewTest do
+  use WandaWeb.ConnCase, async: true
+
+  import Phoenix.View
+  import Wanda.Factory
+
+  alias Wanda.Results.ExecutionResult
+
+  describe "ExecutionView" do
+    test "renders list_checks_executions.json" do
+      [
+        %ExecutionResult{execution_id: execution_1, group_id: group_1},
+        %ExecutionResult{execution_id: execution_2, group_id: group_2},
+        %ExecutionResult{execution_id: execution_3, group_id: group_3}
+      ] = data = build_list(3, :execution_result)
+
+      assert %{
+               data: [
```

Comment:
I don't know if in this assert `data: ^data` would work (I think it should). It is certainly shorter hehe

Reply:
I think you're right, I'll try in another PR so that @fabriziosestito doesn't get blocked too much by this :smile: